tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-    "extends": "./node_modules/pc-nrfconnect-shared/config/tsconfig.json"
+    "extends": "./node_modules/pc-nrfconnect-shared/config/tsconfig.json",
+    "compilerOptions": {
+        "module": "CommonJS"
```

Comment:
A bit curious why this has to be CommonJS

Reply:
It cannot be the default because then it flags [the dynamic import in `devtools.ts`](https://github.com/NordicSemiconductor/pc-nrfconnect-launcher/blob/7406ffdb682515421137d9823c0c09d55dea8c80/src/main/devtools.ts#L83) with

> `TS1323: Dynamic imports are only supported when the '--module' flag is set to 'es2020', 'es2022', 'esnext', 'commonjs', 'amd', 'system', 'umd', 'node16', or 'nodenext'.`

On https://www.typescriptlang.org/tsconfig#module they say: 

> You very likely want `"CommonJS"` for node projects.

But it seems like other options such as `ES2022` would work as well.